routers/web/repo/issue.go

```diff
@@ -413,6 +413,22 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption uti
 		ctx.ServerError("GetPinnedIssues", err)
 		return
 	}
+	//	If Show Archived Label check box is not checked, we don't want to select archived labels
+	// and hence they will be unselected and hide from the list.
+	if !ctx.FormBool("archived") && len(labelIDs) > 0 {
+		var tempLabelIds []int64
+		for _, l := range labels {
+			for _, labelId := range labelIDs {
+				if labelId == l.ID && !l.IsArchived() {
+					tempLabelIds = append(tempLabelIds, labelId)
+				}
+			}
+		}
+		if len(tempLabelIds) > 0 {
+			labelIDs = tempLabelIds
+			selectLabels = strings.Join(base.Int64sToStrings(labelIDs), ",")
+		}
+	}
```

Comment:
Wait, why is this suddenly a backend task again?
I thought we discussed that this is a frontend-only task (where we simply pass through the query param)?

Reply:
This is not needed.